pipelines/core-services/core-services.yaml

```
@@ -219,11 +219,6 @@ spec:
     taskRef:
       name: prefetch-dependencies
       version: "0.2"
-    when:
-    - input: $(params.prefetch-input)
-      operator: notin
-      values:
-      - ""
```

I'll make the comment here then since the other PR was closed as a duplicate. Should we make these conditional operations consistent with the build tasks? i.e. 
```suggestion
    when:
    - input: $(tasks.init.results.build)
      operator: in
      values:
      - "true"
```

---

IIUC the goal is to always run prefetch, so I'm in favor of removing the when condition entirely, (i.e. this PR as it is now), rather than reworking the logic.